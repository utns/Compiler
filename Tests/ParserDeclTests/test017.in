type
    a = 1..'str';