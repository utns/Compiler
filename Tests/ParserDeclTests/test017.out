Incorrect const ''str'': line = 2 pos = 12
